internal/featuretests/v3/featuretests.go

```diff
@@ -110,6 +112,10 @@ func setup(t *testing.T, opts ...interface{}) (cache.ResourceEventHandler, *Cont
 		Builder: dag.Builder{
 			Source: dag.KubernetesCache{
 				FieldLogger: log,
+				ConfiguredGateway: types.NamespacedName{
+					Name:      "contour",
+					Namespace: "projectcontour",
```

Comment:
I thought the ns/name comes from the contour cfgmap?

Reply:
We don't have any good tests that cover configmap --> code (like a feature test). I know @skriss is working on some of those integration test rework which I think will allow us to better validate the settings from the configmap onwards.